ios/sdk/src/picture-in-picture/PiPViewCoordinator.swift

```diff
@@ -109,14 +117,19 @@ public class PiPViewCoordinator {
     /// around screen, and add a button of top of the view to be able to exit mode
     public func enterPictureInPicture() {
         isInPiP = true
+        // Resizing is done by hand when in pip.
+        view.autoresizingMask = []
```

Comment:
I don't like that I have to remove the `autoresizingMask` and add it again when exiting picture in picture mode.
Is there a better solution?